nginx/modsecurity/apr_bucket_nginx.c

```diff
@@ -22,7 +22,6 @@ typedef struct apr_bucket_nginx {
 
 /* ngx_buf_t to apr_bucket */
 apr_bucket * apr_bucket_nginx_create(ngx_buf_t *buf,
-                                     apr_pool_t *p,
```

Comment:
was p unused?

Reply:
Yes, it was unused. Previously, that wasn't a big deal since we just passed a `ngx_http_request_t->pool` inside which resulted in a no-op, but now those helpers are sometimes called in contexts where no `ngx_http_request_t` exists.